sdk/test/UnitTests/AWSSDKHandlerTests.cs

```diff
@@ -131,6 +131,16 @@ public void TestLoadServiceHandlerManifestWithDefaultConfigurationForAWSSDKHandl
             Assert.IsNotNull(handler.AWSServiceHandlerManifest);
         }
 
+        [TestMethod]
+        public void TestLoadServiceHandlerManifestWithDefaultConfigurationForAWSSDKHandlerAsStream()
+        {
+            using (Stream stream = new FileStream(_path, FileMode.Open, FileAccess.Read))
+            {
+                var handler = new XRayPipelineHandler(stream);

```

Comment:
A unit test for null stream would be nice.

Reply:
Added!